src/VisualStudio/Core/Next/Progression/CSharpGraphProvider.cs

```diff
@@ -12,7 +11,7 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.CSharp.Progression
 {
-    [GraphProvider(Name = "CSharpRoslynProvider", IntellisenseType = Guids.CSharpProjectRootIdString)]
+    [GraphProvider(Name = "CSharpRoslynProvider", ProjectCapability = "CSharp")]
```

Comment:
That constant resolves to C# not CSharp which is the project capability. Those constants are defined in CPS but I'm presuming that we don't really want to take a CPS dependency here just for these strings